docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,842 @@
+|ns
+|the total elapsed time of commit operations' executions to the destination endpoint on the latest repartitioning round
+
+|`partitions.elapsedMigrationOperationTime`
+|ns
+|the total elapsed time of migration & replication operations' executions from source to destination endpoints on the latest repartitioning round
+
+|`partitions.elapsedMigrationTime`
+|ns
+|the total elapsed time from start of migration tasks to their completion on the latest repartitioning round
+
+|`partitions.lastRepartitionTime`
+|ms
+|the latest time that repartition took place
+
+|`partitions.localPartitionCount`
+|count
+|the number of partitions currently owned by given member
+
+|`partitions.maxBackupCount`
+|count
+|
+
+|`partitions.memberGroupsSize`
+|count
+|the number of the member groups to be used in partition assignments
+
+|`partitions.migrationActive`
+|boolean
+|the number of active migration tasks
+
+|`partitions.migrationQueueSize`
+|count
+|the number of migration tasks in the migration queue
+
+|`partitions.partitionCount`
+|count
+|total partition count
+
+|`partitions.plannedMigrations`
+|count
+|the number of planned migrations on the latest repartitioning round
+
+|`partitions.replicaSyncRequestsCounter`
+|count
+|the number of replica sync requests
+
+|`partitions.replicaSyncSemaphore`
+|count
+|the permits count of replica sync semaphore
+
+|`partitions.stateStamp`
+|count
+|the stamp value for the current partition table. Stamp is calculated by hashing the individual partition versions using MurmurHash3. If stamp has this initial value, 0L, then that means partition table is not initialized yet.
+
+|`partitions.totalCompletedMigrations`
+|count
+|the total number of completed migrations
+
+|`partitions.totalElapsedDestinationCommitTime`
+|ns
+|the total elapsed time of commit operations' executions to the destination endpoint since the beginning
+
+|`partitions.totalElapsedMigrationOperationTime`
+|ns
+|the total elapsed time of migration & replication operations' executions from source to destination endpoints since the beginning
+
+|`partitions.totalElapsedMigrationTime`
+|ns
+|the total elapsed time from start of migration tasks to their completion since the beginning
+
+|`pnCounter.creationTime`
+|ms
+|the creation time of the PN counter on the member
+
+|`pnCounter.totalDecrementOperationCount`
+|count
+|the number of subtract (including decrement) operations on this PN counter
+
+|`pnCounter.totalIncrementOperationCount`
+|count
+|the number of add (including increment) operations on this PN counter
+
+|`pnCounter.value`
+|count
+|the current value of the pn counter
+
+|`proxy.createdCount`
+|count
+|the number of created proxies for a given service.
+
+|`proxy.destroyedCount`
+|count
+|the number of destroyed proxies for a given service.
+
+|`proxy.proxyCount`
+|count
+|the number of active proxies for a given service. e.g., the number of all proxies for the IMap.
+
+
+|`raft.destroyedGroupIds`
+|count
+|
```

Comment:
Do you need help with these?

Reply:
I need help with them. I can only convert the metric name to the description but this won't be explanatory.